src/test/kotlin/no/nb/mlt/wls/order/controller/OrderControllerTest.kt

```diff
@@ -211,8 +309,9 @@ class OrderControllerTest(
             callbackUrl = "callbackUrl"
         )
 
+    // dop = Duplicate Order Payload
     // Will exist in the database
-    private val duplicateOrderPayload =
+    private val dop =
```

Comment:
If you've changed it to `dop` then we might as well rename `testOrderPayload` to `top` and probably do it in all the test files we have. Also best to add the `// dop = Duplicate Order Payload` as a JavaDoc so that Intellij shows it in case someone forgets what the abbreviation stands for.
I personally prefer to only use abbreviations when names are extra long or the abbreviation is well known, such as `cut = Class Under Test`, as it might make code harder to read and reason about otherwise. I like it when code can be easily read in plain text editor/terminal/etc without any fancy help tools. You might also check with Tom Kristian to see what his opinion is.

Reply:
I made that change one day as I believed it to be inconsistent. This was only true when looking at the service tests (OrderServiceTest and ProductServiceTest). If we do make this change I will ensure everything has relevant JavaDoc, but I do agree that when reading the code for review it would be better if all things were abbreviated

Reply:
After chatting with Tom we agreed on making the code more readable by expanding the abbrevations